src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -33,9 +46,13 @@ export function SwapSettingsSlippageInput({
   // Parses the input and updates slippage if valid
   const handleSlippageChange = useCallback(
     (newSlippage: string) => {
-      const newSlippageNumber = Number.parseFloat(newSlippage);
-      if (!Number.isNaN(newSlippageNumber)) {
-        updateSlippage(newSlippageNumber);
+      if (newSlippage === '') {
+        setSlippage(0);
```

Comment:
A cleaner way to write code, it's to add `return;` if the logic is over.
So that you don't need the final `else`.

Reply:
Nice tip, I like that